GoogleForCreators/web-stories-wp
# Disable height field if unlocked and text is in selection.

## Summary



## Relevant Technical Choices



## To-do

- [ ] Confirm mixed multi-selection behavior.



## User-facing changes
The user can not edit text elements height independently.


## Testing Instructions
I
1. Add a Text element
2. Set the width/height unlocked
3. Verify that the height is displaying "AUTO" and the field is disabled
4. Verify that changing width leaves height the same.
5. Lock width and height
6. Verify that now the height can be changed as well and the width changes together with it.

II
1. Add a Text element and another element (e.g shape)
2. Select both
3. Verify the same behavior as before -- if the height/width are unlocked, only the width can be changed from the input, otherwise both.



---



Fixes #1950 
Fixes #3637
